Don't enforce version for libuuid; ours comes from e2fsprogs while this check is
meant for the one coming from util-linux.

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -120,7 +120,6 @@ set(libical_glib_minimum_version 3.0.7)
 set(libsecret_minimum_version 0.5)
 set(libxml_minimum_version 2.0.0)
 set(sqlite_minimum_version 3.7.17)
-set(uuid_minimum_version 2.0)
 
 # Optional Packages
 set(goa_minimum_version 3.8)
@@ -421,7 +420,7 @@ endif(ENABLE_GTK4)
 # **************************************************************
 
 pkg_check_modules(JSON_GLIB REQUIRED json-glib-1.0>=${json_glib_minimum_version})
-pkg_check_modules(UUID REQUIRED uuid>=${uuid_minimum_version})
+pkg_check_modules(UUID uuid REQUIRED)
 
 if(ENABLE_GTK)
 	add_printable_option(ENABLE_OAUTH2_WEBKITGTK "Enable WebKitGTK gtk3 for built-in OAuth2 authentications" ON)
